.cirrus.yml

```diff
@@ -27,12 +27,12 @@ task:
     gclient sync
 
   matrix:
-    - name: build_and_test_host
+    - name: build_and_test_host_debug
       compile_host_script: |
         cd $ENGINE_PATH/src
         ./flutter/tools/gn --unoptimized --full-dart-sdk
         ninja -C out/host_debug_unopt
-      test_host_script: cd $ENGINE_PATH/src && ./flutter/testing/run_tests.sh host_debug_unopt
+      test_host_script: $ENGINE_PATH/src/flutter/testing/run_tests.py --variant host_debug_unopt
```

Comment:
This LG but maybe we can document how to run the tests somewhere on the wiki, or update the doc if it's already there.